script/RumpelConfig.sol

```diff
@@ -197,6 +207,7 @@ library RumpelConfig {
     address public constant MAINNET_SY_SUSDE_27MAR2025 = 0x3Ee118EFC826d30A29645eAf3b2EaaC9E8320185;
     address public constant MAINNET_SY_SUSDE_29MAY2025 = 0xE877B2A8a53763C8B0534a15e87da28f3aC1257e;
     address public constant MAINNET_SY_WSTUSR_26MAR2025 = 0x6c78661c00D797C9c7fCBE4BCacbD9612A61C07f;
+    address public constant MAINNET_SY_SUSDE_30JUL2025 = 0xF541AA4d6f29ec2423A0D306dBc677021A02DBC0;
```

Comment:
Historically, for SYs, we've also added the token approve/transfer functions. Were you thinking redeem itself was enough, and we should switch to that moving forward for these token types?

Reply:
No, I think we should still add approve and transfer. Updated with this change.